commands/predict.py

```diff
@@ -203,3 +233,7 @@ def main(
                 json_response_array=job_predictions.json_response_array,
                 model_name_array=job_predictions.model_name_array,
             )
+            kili_print(
+                "Predictions sent to kili, you can open the following url to check them out!"
+            )
+            kili_print(f"{api_endpoint[:-21]}/label/projects/{project_id}/explore")
```

Comment:
Cool! `-21` could be replaced by `.replace("substring_to_replace", "")

Reply:
and wondering if there would be a filter to target the assets that have a prediction label.